### DISCUSSION TITLE
Getting uploaded images from enpoint

### DISCUSSION THREAD

**Discussion by VanjaRadovanovic** [directus/directus#3676] [Archive]
I'm running directus locally:
node.js: v14.15.4
npm: v6.14.11
mysql: v8.0.22

And I uploaded images in directus admin panel and I need to query src so I can implement it in my react app but I only get an ID.
On what endpoint do I have to go to receive the image source url? And is it possible to deploy directus on php server without node.js installed?

**Comment by rijkvanzanten** [comment]
> On what endpoint do I have to go to receive the image source url?

The image source can be retrieved through the `/assets` endpoint, so `/assets` + file ID.

> And is it possible to deploy directus on php server without node.js installed?

No, Directus v9 runs on Node, so it's required to use it.

**Comment by benhaynes** [comment]
If you're getting an ID of the image in a response, you can fetch "deeper" relational data to get the source info too. Check out the `fields` parameter in the Docs:

https://docs.directus.io/reference/api/query/fields.html#fields

The PHP version of Directus 9 is still in development and won't be ready for months still.
